### ISSUE TITLE
Application freezes after deleting task

### ISSUE THREAD

**Issue by markl21** [stanford-rc/gcp-cvat-stanford#1]
When deleting a task, the website freezes. Refreshing the page, however, will correctly update the website. @vsoch

**Comment by vsoch** [comment]
Does the task get deleted?

**Comment by markl21** [comment]
Yes, functionally it still works. However, it was hanging for about a minute before I refreshed to fix it.

**Comment by vsoch** [comment]
Could you give me a timestamp for a delete attempt? If this is an issue with cvat itself (as many might be) we need to open an issue over there, and I can look in the server logs for any hints about the request.

**Comment by vsoch** [comment]
Also, since this is in the UI, I'd always have a console open to look for possible UI errors. It could be that the request was fine to the server, but some refresh / load in the browser didn't happen.

**Comment by markl21** [comment]
![Screenshot from 2020-10-01 16-08-45](https://user-images.githubusercontent.com/60868184/94858082-7be44280-0400-11eb-9710-1fb0d29e462d.png) This is what I see before refreshing. There is no response until I refresh. Timestamp is now (13:10 10.01.2020). I'll open a console in a bit. I believe you are right in that, although the server correctly updates, the browser does not "get the message".
